projects/candle/index.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
     pool2: pool2s(stakingContracts, [USDC_CNDL_UNIV2]),
     tvl: (async) => ({}),
   },
+  candle: {
+    staking: stakings(stakingContracts, CNDL),
```

Comment:
can you pass the chain name ('candle') into the stakings and pool2s functions here? at the moment they're calling ethereum

Reply:
we added candle in sdk 2.3.61

Reply:
I'll fix that thank you and yeah the protocol was added i think but not yet the chain

Reply:
I mean we added candle chain in that sdk version, so if you update the sdk in your package.json then run 'node test.js projects/candle/index.js' it should work with 'candle' chain specified

Reply:
> I mean we added candle chain in that sdk version, so if you update the sdk in your package.json then run 'node test.js projects/candle/index.js' it should work with 'candle' chain specified

Yeah it does, but in order for Candle to show as a chain on DefiLlama we have to merge this too I think

Reply:
yeah true but we need this doing first

> can you pass the chain name ('candle') into the stakings and pool2s functions here? at the moment they're calling ethereum